tsconfig.json

```diff
@@ -4,6 +4,7 @@
     "src"
   ],
   "compilerOptions": {
+    "jsx": "react",
```

Comment:
The `jsx` option should be set to `react-jsx` instead of `react` for compatibility with React 17 and newer versions. This allows for the new JSX transform introduced in React 17.